ui/apps/dev-dashboard/src/components/app-routes.tsx

```diff
@@ -65,23 +70,27 @@ const AppRoutes = (props: RootComponentProps) => {
       <Helmet>
         <title>Dev Dashboard | Ethereum World</title>
       </Helmet>
+
       <Routes>
         <Route path="/" element={<Navigate to={routes[DASHBOARD]} replace />} />
+
         <Route path={routes[DASHBOARD]} element={<DevDashboard {...props} />} />
+
         <Route
           path={routes[ONBOARDING]}
           element={
             <DevDashOnboardingIntro
```

Comment:
since you are not reusing this component why not embed all the texts inside it instead of passing them as props?

Reply:
gotcha, I clean that up too. I did for others.